lib/tokenize.cpp

```diff
@@ -8778,6 +8778,22 @@ void Tokenizer::simplifyKeyword()
                     }
                 }
             }
+
+            // noexcept -> noexcept(true)
+            // 3) void f() noexcept; -> void f() noexcept(true);
+            if (Token::Match(tok, ") noexcept [:{;]") ||
```

Comment:
Thought I've done this in the first try, but didn't work. After testing it again, it worked. Seems like the initial problem was a different one.